src/main/java/org/embl/mobie/viewer/projectcreator/ui/ProjectsCreatorPanel.java

```diff
@@ -38,11 +38,11 @@
     private JComboBox<String> groupsComboBox;
     private JComboBox<String> viewsComboBox;
 
-    private int lastImageTypeIndex = 0;
-    private int lastImageDataFormatIndex = 0;
-    private int lastAddMethodIndex = 0;
-    private boolean lastUseDefaultExportSettings = true;
-    private boolean lastExclusive = false;
+    private ProjectCreator.ImageType imageType = ProjectCreator.ImageType.image;
+    private ImageDataFormat imageDataFormat = ImageDataFormat.BdvN5;
+    private ProjectCreator.AddMethod addMethod = ProjectCreator.AddMethod.link;
+    private boolean useDefaultExportSettings = true;
```

Comment:
@K-Meech Does that work? I think you need to make them `private static` otherwise (without `static`) when you close the class it will not remember...

Reply:
This class persists the whole time the project creator is open (as it has all the main ui stuff in it) - so it will only reset when you close the project creator window. I can make them static, if you want it to persist when you close + re-open the project creator (but without closing fiji)

Reply:
OK... I think it does not harm to make them static, what do you think?

Reply:
I made them static now.

Reply:
ok, you can merge it if it is up to me.